site/source/docs/porting/exceptions.rst

```diff
@@ -4,18 +4,9 @@
 C++ exceptions support
 ==============================
 
-C++ exceptions support can be enabled in emscripten using ``-fexceptions``.
-Note that unlike on most platforms, C++ exceptions are disabled by default due
-to their current overhead, see later.
+By default, exception support is disabled in Emscripten to avoid overhead.
```

Comment:
"exception catching is disabled"  ?  Technically exception throwing is enabled.

Reply:
Ah right. I tried to keep it close to the original, but we can update this to be more accurate while at it.

Reply:
Fixed